modules/amq-broker-s2i/added/configure_s2i_files.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+INSTANCE_DIR=$1
+echo "Copying Config files from S2I build"
+cp -v $AMQ_HOME/conf/* ${INSTANCE_DIR}/etc/
```

Comment:
I'm getting a "cp: cannot stat '/opt/amq/conf/*': No such file or directory" in the logs at runtime. I think the conf directory is missing write permissions for the group user.  If this is the issue, we could set the permissions in this modules or the launch modules(in case other modules are going to write to conf), WDYT?

Reply:
Sorry, I meant read permissions, which do exist for the group. Please disregard what I said above, I'm not sure why this is happening yet

Reply:
Thank you.  
I have fixed the permissions in amq-broker-s2i/install.sh